support/build/Makefile.rules

```diff
@@ -271,13 +271,26 @@ $(eval $(call uc,$(1))_EXPORTS += $(wildcard $(IMPORT_BASE)/exportsyms.uk))
 $(eval $(call uc,$(1))_LOCALS  += $(wildcard $(IMPORT_BASE)/localsyms.uk))
 endef
 
+# addlib_tree $1:libname
+define addlib_tree =
+$(eval $(call uc,$(1))_BUILDTREE := y)
```

Comment:
Maybe we want to hide this variable with a `_` prefix. What do you think about naming it `_$(call uc,$(1))_TREEMODE`?

Reply:
Yeah, good idea if the var is never meant to be exposed to the user. However, wouldn't it be cleaner to keep the libname prefix as-is and preceed the variable name itself with an underscore (i.e., `LIBNAME__TREEMODE` rather than `_LIBNAME_TREEMODE`) so that all of LIBNAME's variables are of the form `LIBNAME_*`?

Reply:
Yeah, I think `LIBNAME__TREEMODE` is also a good idea :+1:, makes sense. So far we use `_` prefix to hide internal Make functions and until recently there was an internal variable: `_IMPORT_BASE` but none of them were local to a library. We can consider this as a new case.